.github/workflows/main.yml

```
@@ -46,12 +45,12 @@ jobs:
     - name: 🔍 Type checking
       run: |
         echo "🔍 Running type checker..."
-        mypy backend/src/ --ignore-missing-imports
+        mypy src/ --ignore-missing-imports
         
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)